src/adserver.js

```diff
@@ -0,0 +1,57 @@
+import {formatQS} from './url';
+
+//Adserver parent class
+var adserver = function(attr) {
+  var that = {};
+  that.name = attr.adserver;
+  that.code = attr.code;
+  that.getWinningBidByCode = function() {
+    var bidObject = $$PREBID_GLOBAL$$._bidsReceived.find(bid => bid.adUnitCode === that.code);
+    return bidObject;
+  };
+  return that;
+};
+
+//DFP ad server
+exports.dfpAdserver = function (options, urlComponents) {
```

Comment:
@mkendall07 correct but the design here forced me to export noun. 
So in future we need video with any other adserver like appnexus adserver or pubmatic adserver than we export two more function/classes like 'anAdserver' or 'pubAdserver'